x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/url_state/url_state.spec.ts

```diff
@@ -165,23 +165,23 @@ describe('url state', () => {
     );
   });
 
-  it('sets kql on network page', () => {
+  it.skip('sets kql on network page', () => {
```

Comment:
this is all the tests, i hope they come back soon. they catch bugs when they dont flake :(

Reply:
Me too!!! :(